src/common/Icons.js

```diff
@@ -84,3 +85,4 @@ export const IconMoreHorizontal = makeIcon(Feather, 'more-horizontal');
 export const IconEdit = makeIcon(Feather, 'edit');
 export const IconPlusSquare = makeIcon(Feather, 'plus-square');
 export const IconVideo = makeIcon(Feather, 'video');
+export const IconUserMuted = makeIcon(FontAwesome5, 'user-slash');
```

Comment:
If we end up using a `FontAwesome5` icon, and if you're interested, I have some working code for letting Flow know that the component can take `brand`, `light`, and `solid` props: https://github.com/oblador/react-native-vector-icons/blob/master/FONTAWESOME5.md#usage